business_requirement_crm/wizard/crm_lead_convert_to_requirement.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2019 Tecnativa - Victor M.M. Torres
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class CrmLeadConvertToRequirement(models.TransientModel):
+    _name = "crm.lead.convert.requirement"
+
+    lead_id = fields.Many2one(
+        'crm.lead',
+        string='Lead',
+        domain=[('type', '=', 'lead')])
+    business_requirement = fields.Html('Customer history')
+
+    @api.model
+    def default_get(self, fields):
+        result = super(CrmLeadConvertToRequirement, self).default_get(fields)
+        lead_id = self.env.context.get('active_id')
+        if lead_id:
+            name = self.env['crm.lead'].browse([lead_id]).name
+            result['lead_id'] = lead_id
+            result['business_requirement'] = name
+        return result
+
+    @api.multi
+    def action_lead_to_business_requirement(self):
+        self.ensure_one()
+        # get the lead to transform
+        lead = self.lead_id
+        customer_history = self.business_requirement
+        partner_id = lead.handle_partner_assignation()[lead.id]
+        business_requirement = ""
+        # create new business.requirement
+        if customer_history:
+            business_requirement = customer_history
+        elif lead.description:
+            business_requirement = lead.description
+        elif lead.name:
+            business_requirement = lead.name
```

Comment:
You can do
vals{'bussines_requirement': customer_history or self.lead_id.description or self.lead_id.name or ''